src/app/services/communibee-backend/content/content.ts

```diff
@@ -1,13 +1,19 @@
 import BackendModel from '../backend-model';
 import { Category, CategoryModel } from '../category/category';
 
-export interface Content {
+export interface BasicContent {
```

Comment:
Nice solution! Is it a common practice? Never saw this.

Reply:
what you asking about? the name of the interface?